feat: gedi-calval-copc

jjfrench opened the pull request.

omshinde reviewed (approved):
Thank you for your efforts. Looks good to me.

jjfrench commented:
@omshinde , okay, finally got the environment flattened and we're passing tests! Want to give it one more quick overview?
Main differences are:
- had to add in pc:encoding, the pointcloud extension needs a new release to drop this
- updated tests to reflect STAC creation
- removed venv docker installs, moved everything to the base conda env we're creating

omshinde reviewed (approved):
Thanks @jjfrench ..Looks good. Also, its good to have conda environment working than venv.
